rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -0,0 +1,101 @@
+%global gem_name smart_proxy_openscap
+
+%global foreman_proxy_bundlerd_dir /usr/share/foreman-proxy/bundler.d
+%global foreman_proxy_pluginconf_dir /etc/foreman-proxy/settings.d
+%global spool_dir /var/spool/foreman-proxy/openscap
+%global proxy_user foreman-proxy
+
+Name: rubygem-%{gem_name}
+Version: 0.3.0
+Release: 1%{?dist}
+Summary: OpenSCAP plug-in for Foreman's smart-proxy.
+Group: Applications/Internet
+License: GPLv2+
+URL: http://github.com/openscap/smart_proxy_openscap
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+#Requires: ruby(release)
+Requires: ruby(rubygems)
+Requires: foreman-proxy >= 1.7.0-0.develop.201410221520
+Requires: crontabs
+#BuildRequires: ruby(release)
+BuildRequires: rubygems-devel
+BuildRequires: ruby
+BuildArch: noarch
+Provides: rubygem(%{gem_name}) = %{version}
+Obsoletes: rubygem-foreman-proxy_openscap
```

Comment:
best to put a version in too, to avoid problems later if it changes, e.g. `Obsoletes: foo < 0.3.0-1`